src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -0,0 +1,34 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.jetbrains.annotations.NotNull;
+
+import javax.persistence.*;
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity(name = "CARTS")
+public class Cart {
+
+    @Id
+    @GeneratedValue
+    @NotNull
+    @Column(name = "CART_ID", unique = true)
+    private Long cartId;
+
+    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
FetchType.EAGER for the userId attribute's @OneToOne annotation might cause performance issues if there are many carts in the system. Consider using FetchType.LAZY instead